src/js/msp/MSPHelper.js

```diff
@@ -559,7 +559,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
 
             case MSPCodes.MSP_DEBUG:
                 for (let i = 0; i < 4; i++)
```

Comment:
There are now 8 debug items instead of 4

Reply:
actually, we should the debug_count from the DEBUG_CONFIG message

Reply:
> 

* debug items added

Reply:
_actually, we should the debug_count from the DEBUG_CONFIG message_

Sorry what was the plan here?